### ISSUE TITLE
[5.0] Fix NativeWindow and AutoLoadBindings

### ISSUE THREAD

**Issue by NogginBops** [opentk/opentk#1667]
### Description

Currently `AutoLoadBindings = true` results in an exception being thrown.

**Comment by Lancelotbronner** [comment]
This broke autoloading on macOS, I get the following error:

```
"OpenTK tried to auto-load the OpenGL bindings. We found the OpenTK.Graphics.OpenGL.GL class, but we could not find the 'LoadBindings' method. If you are trying to run a trimmed assembly please add a [DynamicDependency()] attribute to your program, or set NativeWindowSettings.AutoLoadBindings = false and load the OpenGL bindings manually."
```

**Comment by NogginBops** [comment]
Yeah this is still an issue in 5.0-pre9, will be fixed in 5.0-pre10.
I've not had the time yet to fix and release pre10 yet.

The workaround is to disable auto load and call `GLLoader.LoadBindings(new GLFWBindingsContext())` in OnLoad.
